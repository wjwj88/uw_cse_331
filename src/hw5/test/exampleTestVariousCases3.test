# generic version-Jun Wang 5:00
# Sample testing case3-mulplitle edges from a node.
# Tests that a graph can be created, a pair of nodes and
# an edge between them can be added, and a pair of nodes
# and a single edge can be printed out.


# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4

# Add edges
AddEdge graph1 n1 n2 e1
AddEdge graph1 n2 n1 e2
AddEdge graph1 n1 n4 e3
AddEdge graph1 n1 n3 e6

# Print the nodes in the graph and the outgoing edges from n1, n2
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2