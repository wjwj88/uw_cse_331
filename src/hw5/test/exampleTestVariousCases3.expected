# generic version-Jun Wang 5:00
# Sample testing case3-mulplitle edges from a node.
# Tests that a graph can be created, a pair of nodes and
# an edge between them can be added, and a pair of nodes
# and a single edge can be printed out.


# Create a graph
created graph graph1

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1

# Add edges
added edge e1 from n1 to n2 in graph1
added edge e2 from n2 to n1 in graph1
added edge e3 from n1 to n4 in graph1
added edge e6 from n1 to n3 in graph1

# Print the nodes in the graph and the outgoing edges from n1, n2
graph1 contains: n1 n2 n3 n4
the children of n1 in graph1 are: n2(e1) n3(e6) n4(e3) 
the children of n2 in graph1 are: n1(e2)